exercises/high-scores/example.jl

```diff
@@ -0,0 +1,15 @@
+function scores(a::AbstractArray; latest = false, sorted=false, top=0)
+    if top > 0
+        top > length(a) ? sorted ? (return sort(a)) : (return reverse(sort(a))) : nothing
+        if top == 1
+            return sort(a)[length(a)]
+        end
+        if sorted
+            return (sort(a)[length(a)-top+1:length(a)])
+        else
+            return reverse(sort(a)[length(a)-top+1:length(a)])
+        end
+    end
+
+    latest ? (return a[length(a)]) : return a

```

Comment:
```suggestion
    latest ? (return a[end]) : return a
```

Reply:
You can use `end` for the last index of an array. `end + 1` and similar also work.